config/cms.php

```diff
@@ -93,6 +93,19 @@
 
     'backendSkin' => 'Backend\Skins\Standard',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Determines if logging in backend should run UpdateManager
+    |--------------------------------------------------------------------------
+    |
+    | If value is true it runs UpdateMananger after login in backend.
+    | It's recommended to set this value to 'false' in production enviroments
+    | because it cleares cache on every login in backend.
+    |
+    */
+
+    'runUpdateManagerAfterBackendLogin' => true,
```

Comment:
@Samuell1 That might be an unnecessarily large setting name. Could it be shrunk to something like `updatePluginsOnLogin`?
```suggestion
    'updatePluginsOnLogin' => true,
```

Reply:
@bennothommo I think it will be good to name it like that because updatemanager doesnt only updates plugins but everything.

Reply:
@Samuell1 What about `runMigrationsOnLogin`?